gemrb/GUIScripts/bg2/GUICG4.py

```diff
@@ -49,30 +49,30 @@ def CalcLimits(Abidx):
 	Minimum = 3
 	Maximum = 18
 
-	tmp = Abclasrq.GetValue(KitIndex, Abidx)
-	if tmp!=0 and tmp>Minimum:
-		Minimum = tmp
-
 	Race = Abracerq.GetRowIndex(RaceName)
 	tmp = Abracerq.GetValue(Race, Abidx*2)
-	if tmp!=0 and tmp>Minimum:
+	if tmp != 0:
 		Minimum = tmp
 
 	tmp = Abracerq.GetValue(Race, Abidx*2+1)
-	if tmp!=0 and tmp>Maximum:
+	if tmp != 0:
 		Maximum = tmp
 
+	Add = Abracead.GetValue(Race, Abidx)
 	Race = Abracead.GetRowIndex(RaceName)
```

Comment:
These two lines look like they might need to be swapped?

Reply:
The two tables have the same rows, so in practice there's no difference. But better be consistent, I agree.